airflow-core/src/airflow/api_fastapi/common/types.py

```diff
@@ -96,3 +96,10 @@ class MenuItem(Enum):
     PROVIDERS = "Providers"
     VARIABLES = "Variables"
     XCOMS = "XComs"
+
+
+class UIAlert(BaseModel):
+    """Optional alert to be shown at the top of the page."""
+
+    text: str
+    category: Literal["info", "warning", "error"] | None = None
```

Comment:
```suggestion
    category: Literal["info", "warning", "error"] = "info"
```

So we don't have to handle the `None` case on the front-end.

Reply:
Ack, will do